src/patternfly/components/Badge/badge.hbs

```diff
@@ -1,3 +1,6 @@
-<span class="pf-c-badge {{ pf-c-badge--modifier }}">
+<span class="pf-c-badge{{#if badge--modifier}} {{badge--modifier}}{{/if}}"
+  {{#if badge--attribute}}
+    {{{ badge--attribute}}}
+  {{/if}}>
   {{> @partial-block}}
-</span>
+</span>
```

Comment:
this file needs a blank line at the end. if you look at the badge examples in the workspace, there is no space between them.

Reply:
fixed

Reply:
Have you pushed this update up? Still not seeing the blank line in the hbs file, or space between the badges visually in the workspace examples.

Reply:
sorry just pushed it, i see the spacing between the badges now